cypress/integration/specialFeatures/scriptSwitching/testsForAMPOnly.js

```diff
@@ -0,0 +1,7 @@
+// As this e2e user journey revolves around page navigation and cookies preferences, AMP testing is not relevant
+// When you navigate from an AMP page to another - you always progress onto a canonical url
+// AMP cookies are saved in a different way to canonical - so there is no persistance of cookeis between AMP & canonical
```

Comment:
Teeny typo - cookies

Reply:
Given that we're no longer asserting on cookie values, should we just remove this?

Reply:
gid spoat